keyform/management/commands/email.py

```diff
@@ -0,0 +1,54 @@
+from django.core.management.base import BaseCommand
+from keyform.models import Request, Contact, Building
+from django.core.mail import send_mail
+from django.core.mail import EmailMultiAlternatives
+from django.template.loader import render_to_string
+from django.utils.html import strip_tags
+from django.conf import settings
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **kwargs):
+        send_locksmith_emails()
+        send_update_emails()
+
+
+def send_locksmith_emails():
+    email_requests = Request.objects.filter(locksmith_email_sent=False)
+    buildings = Building.objects.all().prefetch_related('contact_set')
+
+    recipient_dict = {b:b.contact_set.all() for b in buildings}
+
+    for request in email_requests:
+        recipients = recipient_dict[request.building]
+        subject = settings.EMAIL_SUBJECT_PREFIX + ': A request has been created'
```

Comment:
Don't prefix a colon (:) to your subject line. That's up to the EMAIL_SUBJECT_PREFIX to include that or not. Don't include a space either.
Also this should be translated.